sdk/src/cache/cache_client.rs

```diff
@@ -76,12 +77,14 @@ use crate::{utils, IntoBytes, MomentoResult};
 /// ```
 #[derive(Clone, Debug)]
 pub struct CacheClient {
-    pub(crate) data_client: ScsClient<InterceptedService<Channel, HeaderInterceptor>>,
+    pub(crate) data_clients: Vec<ScsClient<InterceptedService<Channel, HeaderInterceptor>>>,
     pub(crate) control_client: ScsControlClient<InterceptedService<Channel, HeaderInterceptor>>,
     pub(crate) configuration: Configuration,
     pub(crate) item_default_ttl: Duration,
 }
 
+static NEXT_DATA_CLIENT_INDEX: AtomicUsize = AtomicUsize::new(0);
```

Comment:
@kvcache let me know if there's a better way to do this.

Reply:
There are other ways but it probably doesn’t matter for what you’re doing here